mm2src/coins/Cargo.toml

```diff
@@ -117,6 +118,8 @@ spl-token = { version = "3", optional = true }
 spl-associated-token-account = { version = "1", optional = true }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
+ff = "0.8"
```

Comment:
Can be removed
```suggestion
```

Reply:
done

Reply:
They are different dependencies?

Reply:
Aah, I see, `from_raw` is the trait from `ff`.